README.md

```diff
@@ -169,7 +169,7 @@ yarn link matrix-js-sdk
 yarn install
 ```
 
-See the [help for `yarn link`](https://classic.yarnpkg.com/docs/cli/link) for
+See the [`help for yarn link`](https://classic.yarnpkg.com/docs/cli/link) for
```

Comment:
`yarn link` is a command. That is why it was in backticks. Can you roll this change back?

```suggestion
See the [help for `yarn link`](https://classic.yarnpkg.com/docs/cli/link) for
```